OpenOversight/app/__init__.py

```diff
@@ -99,6 +99,14 @@ def get_age_from_birth_year(birth_year):
         if birth_year:
             return int(datetime.datetime.now().year - birth_year)
 
+    @app.template_filter("field_in_query")
+    def field_in_query(form_data, field):
+        """
+        Determine if a field is specified in the form data, and if so return a Bootstrap
+        class which will render the field accordion open.
+        """
+        return " in " if form_data[field] else ""
```

Comment:
Is this safe? Or should it be:

```suggestion
        return " in " if field in form_data else ""
```

Or maybe it needs to check the value too... in which case:

```suggestion
        return " in " if form_data.get(field) else ""
```

But not sure which makes the most sense.

Reply:
Oh that first suggestion is perfect!

Reply:
Ah, turns out it needed to be the latter!